src/test/java/com/greenfox/lvlup/controller/PitchControllerTest.java

```diff
@@ -0,0 +1,53 @@
+package com.greenfox.lvlup.controller;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.greenfox.lvlup.model.MockingElements;
+import com.greenfox.lvlup.model.PitchSetDTO;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.MediaType;
+import org.springframework.test.context.junit4.SpringRunner;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.*;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.content;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.jsonPath;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@RunWith(SpringRunner.class)
+@WebMvcTest(PitchController.class)
+public class PitchControllerTest {
+
+  @Autowired
+  MockMvc mockMvc;
+
+  String token = "testToken";
+  MockingElements mockingElements = new MockingElements();
+
+  PitchSetDTO pitchSetDTO = new PitchSetDTO();
+
+  @Test
+  public void getPitchesWithCorrectHeader() throws Exception {
+    mockMvc.perform(get("/pitches")
+        .header("userTokenAuth", token)
+        .contentType(MediaType.APPLICATION_JSON))
+        .andExpect(status().isOk())
+        //.andExpect(content().json(mockingElements.getSucessful()))
```

Comment:
We shouldn't leave comments in the code, and also this commented out line is not needed anymore, so you could remove it  ? :)